common/djangoapps/student/models.py

```diff
@@ -1320,7 +1312,7 @@ def username(self):
 
     @property
     def course(self):
-        return modulestore().get_course(self.course_id)
+        return CourseOverview.get_from_id(self.course_id)
 
```

Comment:
Additionally, the `.course_overview` method is more performant because it stores the CourseOverview in a private variable after the first call to it. So if you must keep `.course`, it'd be best to just make it call `.course_overview`